packages/block-library/src/embed/test/index.js

```diff
@@ -75,13 +93,21 @@ describe( 'utils', () => {
 	} );
 	describe( 'createUpgradedEmbedBlock', () => {
 		describe( 'do not create new block', () => {
-			it( 'when block type does not exist', () => {
+			it( 'when block variation does not exist', () => {
```

Comment:
The previous test should not be removed ( `it( 'when block type does not exist', () => {...` ).

It tests the case of `core/embed` block being registered, in utils line 100.
```
if ( ! url || ! getBlockType( DEFAULT_EMBED_BLOCK ) ) return;
```

Reply:
d80d2c38b8113a8b11fa698a8b65453213f192ad